test-packages/e2e-tests/srv/test-service.cds

```diff
@@ -21,4 +21,6 @@ service TestService {
   action createTestEntityById(id : Integer) returns TestEntity;
   action createTestEntityByIdReturnId(id : Integer) returns Integer;
   action createTestEntityReturnId(param : TestEntity) returns Integer;
+
+  entity TestEntity50Col as projection on cloudsdk.TestEntity50Col;
```

Comment:
Minor: 50Col = 50 properties? Maybe there is a better name out there (but maybe not ;) )